d2core/d2ui/scrollbar.go

```diff
@@ -51,8 +51,8 @@ func (v Scrollbar) getBarPosition() int {
 	return int((float32(v.currentOffset) / float32(v.maxOffset)) * float32(v.height-30))
 }
 
-func (v *Scrollbar) Activate() {
-	_, my := ebiten.CursorPosition()
+func (v *Scrollbar) OnMouseMove(event d2input.MouseMoveEvent) bool {
```

Comment:
I haven't tested this change yet

Reply:
Just create a bunch of characters and it should activate the scrollbar.